src/measure_theory/measure_space.lean

```diff
@@ -2613,4 +2614,9 @@ lemma cond {A B : set α} (hA : measurable_set A) (hB : measurable_set B)
   {i : bool} : measurable_set (cond i A B) :=
 by { cases i, exacts [hB, hA] }
 
+@[measurability]
+lemma symm_diff {A B : set α} (hA : measurable_set A) (hB : measurable_set B) :
+  measurable_set (A Δ B) :=
+(hA.diff hB).union (hB.diff hA)
+
```

Comment:
To me this feels like it would make more sense here:

https://github.com/leanprover-community/mathlib/blob/9aa4c10e5088348952a923a0e2cbd5ffef27680e/src/measure_theory/measurable_space_def.lean#L188-L191

Is there a reason you chose not to put it there?

Reply:
(`measurable_set.diff` possibly should be named `measurable_set.sdiff`, but maybe that's for a separate PR.)

Reply:
If I put it in `measurable_space_def` then I can't use the measurability tag

Reply:
That's fine, `diff` doesn't have the measurability tag either - you can add the tag in the same place that `diff` earns it.

Reply:
See also #8238

Reply:
https://github.com/leanprover-community/mathlib/blob/3870896acc8abddf1712ab190007beede569e049/src/measure_theory/tactic.lean#L55

Reply:
> See also #8238

I think that also belongs in this file.

Reply:
Alright, I will move them over